avantis_trader_sdk/rpc/fee_parameters.py

```diff
@@ -28,7 +29,16 @@ async def get_margin_fee(self):
         """
         Multicall = self.client.contracts.get("Multicall")
         pairs_info = await self.client.pairs_cache.get_pairs_info()
-        raw_data = await Multicall.functions.getMargins().call()
+
+        try:
+            raw_data = await Multicall.functions.getMargins().call()
+            # Log the raw data to inspect it
+            print(f"Raw data from getMargins(): {raw_data}")
```

Comment:
is this for debugging?

Reply:
Previously getMargin() function throwing error, to log the error used that, as in Latest PR contracts updated getMargin function working properly.